R/indexing.R

```diff
@@ -1,3 +1,28 @@
+is_pure_fancy_indexing <- function(selection, ndim = length(selection)) {
```

Comment:
Would you be able to add a test for this function and documentation comments to convey the purpose?

Reply:
documentation 0416209f4f3db0c964eff2204b9519193c7a7d70